### DISCUSSION TITLE
v2.5.5

### DISCUSSION THREAD

**Discussion by mashb1t** [lllyasviel/Fooocus#3508] [General]
## What's Changed
* fix: resolve colab unsupported image type issue by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3506

**Full Changelog**: https://github.com/lllyasviel/Fooocus/compare/v2.5.4...v2.5.5

<hr /><em>This discussion was created from the release <a href='https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.5'>v2.5.5</a>.</em>

**Comment by maxmuk1** [comment]
What are source codes? are we meant to install them each time?

**Reply by mashb1t** [reply]
What are you referring to? Please elaborate.

**Reply by maxmuk1** [reply]
I'm referring to the fact that there is an updated source code every few weeks... Just like right now this is v2.2.5

As a complete beginner, I'm wondering if we're supposed to download and install them each time, what their purpose is how to install them appropriately. 

If you would be so kind as to inform me I would be very grateful. 

Thanks Mash

**Reply by mashb1t** [reply]
Fooocus auto-updates when you start it, please check the logs. I doubt that there will be frequent updates in the future, see https://github.com/lllyasviel/Fooocus/discussions/3504 and readme ("security support")

**Reply by maxmuk1** [reply]
I'm getting a ModuleNotFoundError: No Model Named "Supervision" 

Which means, the program isn't opening up...

Any help would be appreciated. 

Thanks Mash

**Reply by maxmuk1** [reply]
![Uploading Screenshot 2024-08-23 183434.png…]()

**Reply by mashb1t** [reply]
Please find this information in the discussions/issues, e.g. https://github.com/lllyasviel/Fooocus/discussions/3284.

**Reply by Deij123** [reply]
> Эту информацию можно найти в обсуждениях/вопросах, например, #3284 .

You once said that new drivers don’t work well and the old 531 ones can no longer be installed

**Reply by Deij123** [reply]
> Please find this information in the discussions/issues, e.g. #3284.

Hello, generating one image takes 7 minutes with stock settings. i5 13400 gtx4060 8gb ram 16gb ddr5 ssd m2. 2 tr Is this normal?

**Reply by Rogala** [reply]
Install the latest drivers, 40GB swap file, 32GB RAM preferably

7 minutes is a long time, should be about a minute, with preset Quality and a couple of Loras

**Comment by Kira8822** [comment]
What do you think about this fooocus fork that has the possibility of working calmly with Flux? 
https://github.com/metercai/SimpleSDXL 👈🏻

**Reply by DavidDragonsage** [reply]
Yes, SimpleSDXL2 works very well with Flux. I use this combination a lot.

Edited to add: since I wrote this recommendation I discovered in December that SimpleSDXL2 has a number of serious security concerns. I am now recommending [RuinedFooocus](https://github.com/runew0lf/RuinedFooocus) instead for those interested in working with Flux in a Fooocus environment.

In addition, I am actively developing an independent fork, [FooocusPlus](https://github.com/DavidDragonsage/FooocusPlus), that solves all of the security issues in SimpleSDXL2, supports Flux and other large models, while including _all_ the features of the original Fooocus environment. We are currently in the pre-release stage and plan to be ready for full release in a few weeks.

**Reply by badraymen** [reply]
@DavidDragonsage youppppuiiiii !!! Very good news !! Any links or page for FooocusPlus??

**Reply by mashb1t** [reply]
@badraymen idk, have you tried... clicking(!) on one of the links above? 🤯

**Comment by MFallal** [comment]
Hi Guys, when I run it and download all but when it finishes nothing run ?

**Reply by Rogala** [reply]
By specifying the parameters and OS of your PC you will make it easier for us to help you

**Reply by MFallal** [reply]
System:64-bit operating system, x64-based processor
Processor: Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz   2.30 GHz
RAAM: 16.0 GB (15.8 GB usable)

**Reply by Rogala** [reply]
It's a laptop with no discrete video card, weak CPU, little RAM, you can't even run Fooocus on it in CPU mode.

**Reply by MFallal** [reply]
hahahha, okay thank you very much

**Comment by khushij44** [comment]
Hello,

Once the download file available under version 2.5.0 was downloaded, it says the file is ''Failed - Needs authorization''
How can I Install please?

**Comment by badraymen** [comment]
it is the end of devlopping fooocus ? flux killed XL !! i dont like to go tu webUI forge or comfyUi :'(

**Reply by mashb1t** [reply]
see https://github.com/lllyasviel/Fooocus/discussions/3840#discussioncomment-11860334

**Reply by DavidDragonsage** [reply]
@badraymen Also see my note about nine posts up.

**Comment by PPMSol** [comment]
Hello and thank you for the amazing work on Fooocus!

I noticed that in the latest release (v2.5.5) there is no Windows-ready build (such as Fooocus-windows.zip or Fooocus-win-v2.5.5.7z) that includes the run.bat file. The only available downloads are the source code archives.

Would it be possible to upload the prebuilt Windows package (with run.bat included) so that non-developers can launch Fooocus more easily?

Thank you in advance for your support and for making this tool accessible 🙏
Best regards,
Kanat

**Reply by Rogala** [reply]
[2.5.0](https://github.com/lllyasviel/Fooocus#download)

2.5.0 is automatically updated on startup

**Comment by AlanDeor** [comment]
Hello! Please tell me when to expect an update for the 5060 series video cards. I have similar errors on all neural networks and I can't generate them, nothing happens.

Processor: AMD Ryzen 7500f
— Card: Palit GeForce RTX 5060 Ti Infinity 3 at 16 GB
— DDR5: ADATA XPG at 32 GB
— M2 SSD drive: ADATA at 1 TB
— M2 SSD drive: Kingston at 1 TB
— Board: MSI PRO B650-A
— PSU: Deepcool PL at 800W
— Cooling: ID-Cooling FX360
— Case: XASTRA Q700
— RGB fans: 10 pieces
— Additional: Wi-Fi and Bluetooth

as I understand it's all because of the new Nvidia drivers 580 and Сuda 13.

в Automatic1111 a similar problem, after trying to generate something it says an error.

RuntimeError: CUDA error: no kernel image is available for execution on the device CUDA kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect. For debugging consider passing CUDA_LAUNCH_BLOCKING=1. Compile with TORCH_USE_CUDA_DSA to enable device-side assertions.

Please help me solve the problem.
I'm not as smart as you, I don't understand much about this and I would like to just click the install button and have everything work. I really count on you.

**Reply by Rogala** [reply]
[No problems at all, everything works fine, take a look here](https://github.com/Rogala/AI_Attention)
<img width="923" height="427" alt="image" src="https://github.com/user-attachments/assets/ef5b9581-73a8-4769-a5ef-681b0cf834f8" />

**Reply by AlanDeor** [reply]
oh, What am I doing wrong then? (I installed everything, but after pressing the generate button nothing happens. ((

**Reply by AlanDeor** [reply]
@Rogala Thanks for the link, I will study it. Thank you.

**Reply by AlanDeor** [reply]
oh my god, this is all so complicated, I'm so stupid that I didn't understand anything. It looks like I'm not destined to use AI on my computer and will have to pay for subscriptions.

**Comment by DavidDragonsage** [comment]
@AlanDeor  You could try either [RuinedFooocus](https://github.com/runew0lf/RuinedFooocus) or [FooocusPlus](https://github.com/DavidDragonsage/FooocusPlus). They do support NVidea 5060 right out of the box and they are more or less similar to original Fooocus. FooocusPlus is much closer to original Fooocus but I must confess right away that I am one of its developers. 🙂

**Reply by AlanDeor** [reply]
that's great! is it bad that you are a developer?  
I'll try! I hope I can do it without dancing with a tambourine)

**Comment by thuggun7777m-code** [comment]
after download the latest version for window, run.bat file doesn't show up

**Reply by mashb1t** [reply]
@thuggun7777m-code please make sure to check your firewall/antivirus software for removed files, it might have been deleted this way.

**Reply by marouk12** [reply]
@mashb1t i have the same problem, im click off antivirus and didnt see run.bat :/

**Reply by mashb1t** [reply]
@thuggun7777m-code @marouk12 feel free to create the `run.bat` file yourself, content:

default
```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py
pause
```

realistic
```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset realistic
pause
```

anime
```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset anime
pause
```

**Reply by thuggun7777m-code** [reply]
i dont know how create run,bat file
pls explain easily

**Reply by dnuttz10** [reply]
Help~ is it possible to run this setup on macbook air

**Comment by gutsalovpavel** [comment]
No updates for 1.5 years. Are you still developing the tool?

**Comment by wasallesbelegt** [comment]
<img width="941" height="608" alt="Bildschirmfoto 2026-03-11 um 08 01 09" src="https://github.com/user-attachments/assets/b11d0145-cd12-4b50-8b2a-230705c27a3e" />

So I can’t use it anymore since the update… the installation fails on Colab.

**Comment by DavidDragonsage** [comment]
@gutsalovpavel @wasallesbelegt Just a reminder that you could try either [RuinedFooocus](https://github.com/runew0lf/RuinedFooocus) or [FooocusPlus](https://github.com/DavidDragonsage/FooocusPlus). They do support NVidea 5060 right out of the box and they are more or less similar to original Fooocus. FooocusPlus is much closer to original Fooocus. I'm pretty sure that RuinedFooocus supports Colab. Both of these alternatives are still under active development and they support Flux and other advanced models.

**Comment by b2-mckin** [comment]
It only works like this for me:
!pip uninstall -y numpy cupy-cuda12x cupy
!pip install numpy==1.26.4 cupy-cuda12x
----restart----
%cd /content
!rm -rf Fooocus

!pip install pygit2==1.15.1
%cd /content
!git clone https://github.com/lllyasviel/Fooocus.git
%cd /content/Fooocus
!python entry_with_update.py --share --always-high-vram
